src/main/webapp/script.js

```diff
@@ -22,19 +22,34 @@ function getComments() {
       
       // Display the maximum number of comments
       for (let i = 0; i < data.length; i++) {
-          // Format i comments in a list and display to the webpage.
-          if (data[i] != "") {
-              comments.appendChild(createListElement(data[i]));
-            }
+        // Format i comments in a list and display to the webpage.
+        if(i%3 == 0) {
+          // Add a return between every comment
```

Comment:
A more appropriate comment may be "Add a new line after every comment". This may match the code more too.